Core/scripts/dirac-install.py

```diff
@@ -1112,15 +1112,21 @@ def getExtenalsVersion(self, release=None):
 
   def getDiracOSVersion(self, diracOSVersion=None):
     """
-    It returns the DIRACOS version
-    :param str diracOSVersion: the OS version
-    """
+      It returns the DIRACOS version
+      :param str diracOSVersion: the OS version
+      """
 
     if diracOSVersion:
       return diracOSVersion
     try:
-      return self.prjRelCFG[self.projectName][cliParams.release].get(
+      diracOSVersion = self.prjRelCFG[self.projectName][cliParams.release].get(
           "Releases/%s/DIRACOS" % cliParams.release, diracOSVersion)
+      if not diracOSVersion:
```

Comment:
I do not understand why we need that ? It was not done for Externals or LcgBundle, so why do you need it here ?

Reply:
Because we defined also in release.cfg: https://gitlab.cern.ch/lhcb-dirac/LHCbDIRAC/blob/master/LHCbDIRAC/releases.cfg#L15

Reply:
Let me more precise we take the Externals alway from DIRAC: https://github.com/DIRACGrid/DIRAC/blob/integration/Core/scripts/dirac-install.py#L1109
The LCGVer is taken from the extension: https://github.com/DIRACGrid/DIRAC/blob/integration/Core/scripts/dirac-install.py#L1136

So if I remove https://gitlab.cern.ch/lhcb-dirac/LHCbDIRAC/blob/master/LHCbDIRAC/releases.cfg#L15 LcgVer, it will impossible to install the current version of LHCbDIRAC
So, If we will not have an LHCb specific DIRACOS, we should use from DIRAC. I do not want to specify DIRACOS in LHCbDIRAC, if it is same as in DIRAC...

Reply:
OK I understand